modules/twinkleprotect.js

```diff
@@ -1160,7 +1160,13 @@ Twinkle.protect.callback.evaluate = function twinkleprotectCallbackEvaluate(e) {
 						thispage.setEditProtection(input.editlevel, input.editexpiry);
 					}
 					if (input.movemodify) {
-						thispage.setMoveProtection(input.movelevel, input.moveexpiry);
+						// Ensure a level has actually been chosen
+						if (input.movelevel) {
```

Comment:
I think it should be
```javascript
if (input.movelevel !== 'all') {
```
Because of
https://github.com/azatoth/twinkle/blob/3c65fa5bf409143b99f0af3f4bddbb32a65b45ab/modules/twinkleprotect.js#L388-L392

Reply:
Sorry @Xi-Plus; I've (clearly) already proven to be beyond tired today, so I must be missing what you're saying, but what's the issue with `all`?  It's a valid level for unprotection, becoming `move=all`

Reply:
I wanna know how to trigger the alert (You must chose a move protection level).

Reply:
1. Start with move protection disabled and edit protection selected at the autoconfirmed level
2. Select move protection.  The expiry would be set, but no value for move protection would be selected.
3. Uncheck edit protection.

The form would let you submit and then promptly fail.

Reply:
Thanks a lot!